### PR TITLE
Max file size issue fix

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #150 

Screenshots for the change: 

screenshot_2017-03-19-15-36-45

### CODE REVIEW

**Comment by Abhi2424shek** [comment]
Can we have a text box along with them where the user can enter his choice?

**Comment by rohanagarwal94** [comment]
@Abhi2424shek I guess that will make a simple task complicated. I dont think user needs to put in specific size. @pa1pal thoughts?

**Comment by pa1pal** [comment]
I agree a quick selection option would be better. But using spinner here would be good choice instead of radio buttons in dialog.

**Comment by rohanagarwal94** [comment]
@pa1pal That will interfere with the basic UI of the settings fragment since majorly all the preferences will have dialog. Moreover it would hide the delete the photos preference which would look odd. Suggest otherwise if you still think that would work. Will update the PR then.

screenshot_2017-03-16-22-27-11

**Comment by Abhi2424shek** [comment]
We would go with this then.

File: app/src/main/java/vn/mbm/phimp/me/Settings.java [hunk 1]
```diff
@@ -155,8 +159,10 @@
 	File tmp_folder;
 	int error_count = 0;
 	ProgressDialog pro_gress;
+    private AlertDialog maxSizeDialog = null;
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Settings.java]: please indent whole code.

**Comment by rohanagarwal94** [comment]
@pa1pal made the necessary changes. Please review.

**Pull request merged**